bin/sentieon-genomics-202010.02/lib/python/sentieon/vcflib/vcf.py

```
@@ -0,0 +1,465 @@
+import io
+import itertools
+import os
+import re
+import sys
+import tempfile
+
+from . import bgzf
+from . import sharder
+from . import tabix
+from . import tribble
+from .compat import *
```

Ruff (0.8.2)

`from .compat import *` used; unable to detect undefined names

(F403)